_data/success-stories.yml

```diff
@@ -0,0 +1,80 @@
+cases:
+  - name: Netflix
+    link: https://medium.com/netflix-techblog/global-continuous-delivery-with-spinnaker-2a6896c23ba7
+    logo: cases-logos/netflix.jpg
+    text: “The community uptake has been tremendous! Having a platform that out-of-the-box accommodates multiple cloud providers and an extendable model to participate within, means that we are one large community rather than a bunch of micro-communities focused on various forks. This has the benefit that everyone within the community can take advantage of various innovations.”
```

Comment:
@deesprinter Just out of curiosity, where do the quotations come from. In this case, this language is not directly from the linked blog post. Did you contact the user? 
(This is fine, btw, I'm just trying to determine how directly the words come from the quoted user.)